tools/config-gen/params.yaml

```
@@ -37,3 +38,24 @@
   config-path: "debug.log-mutex"
   type: "bool"
   usage: "Print debug messages when a mutex is held too long."
+
+- flag-name: "log-file"
+  config-path: "logging.file-path"
+  type: "resolvedPath"
+  usage: >-
+    The file for storing logs that can be parsed by fluentd. When not provided,
+    plain text logs are printed to stdout.
+
+- flag-name: "log-severity"
+  config-path: "logging.severity"
+  type: "logSeverity"
+  default: "info"
+  usage: "Specifies the logging severity"
```

Assuming this will show up in GCSFuse help, I think we should write following instead
```suggestion
  usage: "Specifies the logging severity expressed as one of [trace, debug, info, warning, error]"
```